### DISCUSSION TITLE
Welcome to alpaca_zorro_plugin Discussions!

### DISCUSSION THREAD

**Discussion by kzhdev** [kzhdev/alpaca_zorro_plugin#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lorenzodipuccio** [comment]
hi guys thanks for all your work!! Does the plugin supports Alpaca Options?

**Reply by kzhdev** [reply]
I'm working on adding options support. Do you want to help to test?

**Reply by ngigliet** [reply]
here to help if still needed

**Reply by kzhdev** [reply]
Options support is released in 1.4.0
